server/route.go

```diff
@@ -1506,6 +1510,10 @@ func (s *Server) routeAcceptLoop(ch chan struct{}) {
 	s.done <- true
 }
 
+func (s *Server) warnOfClusterTLSInsecure() {
```

Comment:
Why a function for one line?

Reply:
Because I want to call this from reload code. Alternatively, could define the warning text as an internal constant (which I could even use in the test)